src/objectivec/oga_named_tensors.mm

```diff
@@ -0,0 +1,22 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#import "error_utils.h"
+#import "oga_internal.h"
+#import "ort_genai_objc.h"
+
+@implementation OGANamedTensors {
+  std::unique_ptr<OgaNamedTensors> _tensor;
+}
+
+- (instancetype)initWithCXXPointer:(std::unique_ptr<OgaNamedTensors>)ptr
+{
+  _tensor = std::move(ptr);
```

Comment:
general: add `if ((self = [super init]) == nil) { return nil; }` to initializers

also in `OGASequences`

Reply:
fixed.